actionview/lib/action_view/helpers/cache_helper.rb

```diff
@@ -176,7 +184,8 @@ def fragment_name_with_digest(name) #:nodoc:
 
       # TODO: Create an object that has caching read/write on it
       def fragment_for(name = {}, options = nil, &block) #:nodoc:
-        read_fragment_for(name, options) || write_fragment_for(name, options, &block)
+        force = options.is_a?(Hash) && options[:force]
```

Comment:
Introducing keyword arguments does break a couple of tests each (test_output_buffer and test_safe_buffer) in actionpack and actionmailer which are calling `fragment_for` directly with arguments that don't make any sense to me. I get why it _might_ be useful to unit-test a private method, but I don't understand the expectations of the tests.

I'll try an put together a test for this piece on my master PR, but I've no idea how to address failing tests of undocumented expectations of a private method 😕